core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -1811,6 +1811,11 @@ public View getView(String name) {
         return viewGroupMixIn.getViews();
     }
 
+    @Exported     //Just to be exported - @Exported on default methods in interface is not supported
```

Comment:
@oleg-nenashev I was thinking about that, but the current implementation does permission checks implicitly as it [uses](https://github.com/jenkinsci/jenkins/pull/2831/files#diff-9767005f02cfa46f2643177f75ec4d8bR70) `getViews()` under the hood to get views which should already returns filtered results.

Do you think there should be a double check on received views?